packages/monitoring/src/helpers/events.ts

```diff
@@ -0,0 +1,37 @@
+import chalk from "chalk";
+import { EventEmitter } from "events";
+
+const events = new EventEmitter();
+type ServiceStatus = { [key: string]: boolean };
+
+const serviceStatusSummary: ServiceStatus = {};
+const ALIVE = chalk.green.bold("Alive");
+const DOWN = chalk.red.bold("Down");
+function logsHandler(msg) {
```

Comment:
won't this duplicate the logs coming from the grid client? I thought we were going to create another sink

Reply:
will add prefix to the event names